abb_irb4400_support/package.xml

```diff
@@ -3,18 +3,17 @@
   <version>0.1.0</version>
   <description>
     <p>
-      ROS Industrial support for the ABB IRB4400 (and variants).
+      ROS Industrial support for the ABB IRB4400/L30.
```

Comment:
Please leave this "(and variants)". It is sort-of standardised across pkgs, and we won't have to change this when we do add more variants of the 4400 to the pkg.